parsers/graphic_field.go

```diff
@@ -0,0 +1,73 @@
+package parsers
+
+import (
+	hx "encoding/hex"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/ingridhq/zebrash/elements"
+	"github.com/ingridhq/zebrash/printers"
+)
+
+func NewGraphicFieldParser() *CommandParser {
+	const code = "^GF"
+
+	return &CommandParser{
+		CommandCode: code,
+		Parse: func(command string, printer *printers.VirtualPrinter) (interface{}, error) {
+			result := &elements.GraphicField{
+				Position: printer.NextElementPosition,
```

Comment:
What about assigning the default compression type `A` here?

Reply:
I thought about it but ZPL specification says that property field has no default value

Reply:
Ok, could you point me to the documentation? I could not fetch the one on `zebra.com`, so I was referring to this one: https://www.servopack.de/support/zebra/ZPLII-Prog.pdf

Reply:
Yes, sure, I mainly use hints that labelary provides
![Screenshot from 2024-03-01 13-26-54](https://github.com/ingridhq/zebrash/assets/78364711/1d7ffc40-c031-4e4b-b3c7-6bf613e1e585)

Reply:
ok